README.md

```diff
@@ -1,2 +1,17 @@
-# GRASS_Web
-Hugo website for GRASS GIS
+# Hugo-based website for GRASS GIS
+
+## How to install
+
+### Debian
+
+    apt install hugo
+
+## How to build
+
+Run in the `grass-website` root directory:
+
+    hugo
+
+Open web site (using Mozilla Firefox in this case):
```

Comment:
Before this I think is better to link public folder it to a web server directory (as suggested by @nbozon), so the links are working better (not yet correctly)
```suggestion
Link the public folder to your web server directory

ln -s `pwd`/public /your/web/server/directory/grass-website
```